lib/index.js

```diff
@@ -58,7 +58,8 @@ var epilogue = {
       pagination: args.pagination,
       updateMethod: this.updateMethod,
       search: args.search,
-      sort: args.sort
+      sort: args.sort,
+      reloadOnCreate: args.reloadOnCreate
```

Comment:
This is only when the model has includes and that the association has changed. So I think it's 	
insignifiant.